File: WoundHealingWebApp/WoundHealingWebApp/src/app/DTOs/StatsDto.ts
import { WoundTypesStats } from "./WoundTypesStats";

export class StatsDto {
    doctorId!: number;
    patientsCnt!: number;
    finishedTreatmentsCnt!: number;
    appointmentsCnt!: number;
    woundsCnt!: number;
    avgTreatmentDays!: number;
    woundTypesStats!: WoundTypesStats[];
}

File: WoundHealingWebApi/WoundHealingWebApi/DTOs/StatsDto.cs
using System.Collections.Generic;

namespace WoundHealingWebApi.DTOs
{
    public class StatsDto
    {
        public int DoctorId { get; set; }
        public int PatientsCnt { get; set; }
        public int FinishedTreatmentsCnt { get; set; }
        public int AppointmentsCnt { get; set; }
        public int WoundsCnt { get; set; }
        public int AvgTreatmentDays { get; set; }
        public List<WoundTypesStats> WoundTypesStats { get; set; }
    }
}